src/main/java/frc/robot/commands/ShootCommand.java

```diff
@@ -6,17 +6,19 @@
 
 import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj2.command.Command;
-import frc.robot.subsystems.Shooter;
-// import frc.robot.subsystens.Intake; TODO Coordinate with intake to import the right subsystem
+import frc.robot.subsystems.*;
```

Comment:
Nit: Do two import statements, one for `Shooter`, one for `Intake` instead of using the `*` here. You really only have the two but it's still useful to point out we only need these. Usually it's like 5+ imports from the same package where I'll throw in a star. This one is DEFINITELY a style thing and the reasons it can be a good idea may not apply here.